ModelCatalogueGenomicsPlugin/test/integration/org/modelcatalogue/gel/export/AbstractRareDiseasesExporterSpec.groovy

```diff
@@ -158,11 +159,14 @@ class AbstractRareDiseasesExporterSpec extends IntegrationSpec {
             create DataElement called '2nd New Phenotype DataElement'                       //NEW_ELEMENT_CREATED
             update 'containment' of 'Phenotype (9) name 1 1' add '2nd New Phenotype DataElement', 'Min Occurs': 0, 'Max Occurs': 2    //RELATIONSHIP_CREATED
                                                                                                         //RELATIONSHIP_METADATA_CREATED * 2
-
+            sleep(1)
```

Comment:
the test expects changelogs to be created in date order - I think what was happening was that if they are created in the same milli - the order they are retrieved may not be the same as expected